documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -214,21 +274,25 @@ different types will get encoded as the event key's payload:
 |Binary  |BinData("a2Fma2E=",0)|`{ "id" : "{\"$binary\" : \"a2Fma2E=\", \"$type\" : \"00\"}" }`
 |==========================================
 
+ifndef::cdc-product[]
 [WARNING]
 ====
-As of Debezium 0.3.0, the Debezium MongoDB connector ensures that all Kafka Connect _schema names_ are http://avro.apache.org/docs/current/spec.html#names[valid Avro schema names]. This means that the logical server name must start with Latin letters or an underscore (e.g., [a-z,A-Z,\_]), and the remaining characters in the logical server name and all characters in the database and collections names must be Latin letters, digits, or an underscore (e.g., [a-z,A-Z,0-9,\_]). If not, then all invalid characters will automatically be replaced with an underscore character.
+As of Debezium 0.3.0, the MongoDB connector ensures that all Kafka Connect _schema names_ are http://avro.apache.org/docs/current/spec.html#names[valid Avro schema names]. This means that the logical server name must start with Latin letters or an underscore (e.g., [a-z,A-Z,\_]), and the remaining characters in the logical server name and all characters in the database and collections names must be Latin letters, digits, or an underscore (e.g., [a-z,A-Z,0-9,\_]). If not, then all invalid characters will automatically be replaced with an underscore character.
```

Comment:
@gunnarmorling Do you think this section could be rewritten or even omitted perhaps?   I don't think its really applicable to be mentioning anything as it related to version 0.3 at this point in master docs meant for 1.x.

Reply:
I think the note itself is good, just the version reference should be removed:

> The MongoDB connector ensures...